corporate/tests/test_stripe.py

```diff
@@ -2814,11 +2838,6 @@ def test_switch_from_monthly_plan_to_annual_plan_for_manual_license_management(
         for key, value in annual_plan_invoice_item_params.items():
             self.assertEqual(invoice_item[key], value)
 
-        with patch("corporate.lib.stripe.BillingSession.invoice_plan") as m:
-            invoice_plans_as_needed(add_months(self.now, 2))
-            # Even annual plans get invoiced monthly for additional licenses.
-            m.assert_called_once()
```

Comment:
We should still verify it ?

Reply:
Yeah, this does seem like it wants to instead verify that invoicing at 2 months results in a certain number of invoices in the `stripe.Invoice.list` query, even if we're not doing deeper validation of those invoices before invoicing at T+13 months.

Reply:
We alredy verify `self.assertEqual(annual_plan.next_invoice_date, add_months(self.next_month, 1))` above and the main reason I removed it since patching `invoice_plan` causes `plan.next_invoice_date` to not update and causes an infinite while loop here:
```python
        while (
            plan.next_invoice_date is not None  # type: ignore[redundant-expr] # plan.next_invoice_date can be None after calling invoice_plan.
            and plan.next_invoice_date <= event_time
        ):
            billing_session.invoice_plan(plan, plan.next_invoice_date)
            plan.refresh_from_db()
```

Reply:
Yeah, we can move the patch, the thought was just that we could call `invoice_plan` twice rather than once, but maybe it doesn't matter -- as long as adjust the assert statements to check the dates on the invoices, which would be a better tweak.